test/jasmine/tests/plot_api_test.js

```diff
@@ -711,8 +711,12 @@ describe('Test plot api', function() {
                 }
             }];
 
+            spyOn(Plots.subplotsRegistry.gl3d, 'plot');
```

Comment:
fyi @mdtusz @monfera quick and dirty trick to test `Plotly.plot` with gl traces in a way that works on CircleCI.